### PR TITLE
fix `fullAddressPart2` state if empty should not show in address line

### PR DESCRIPTION
**Pull request by casz**
we should not always append state as some countries do not have states in the addresses.

### CODE REVIEW

File: src/models/view/identityView.ts [hunk 1]
```diff
@@ -108,8 +108,13 @@ export class IdentityView implements View {
             return null;
         }
         const city = this.city || '-';
-        const state = this.state || '-';
+        const state = this.state;
         const postalCode = this.postalCode || '-';
-        return city + ', ' + state + ', ' + postalCode;
+        let addressPart2 = city
```
Comment (kspearrin) [inline src/models/view/identityView.ts]: Please use line-ending semicolons.

Comment (casz) [reply src/models/view/identityView.ts]: Good habits die hard 😅

**Comment by casz** [comment]
Feel free to squash 🙇

**Comment by casz** [comment]
Many thanks for the quick merge!

**Pull request merged**